test/manual.js

```diff
@@ -0,0 +1,16 @@
+var resolve = require('../lib');
+
+function main() {
+  var target = process.argv[2];
+  var options = process.argv[3] || '{}';
+  options = JSON.parse(options);
+
+  resolve(target, options).then(function (result) {
+    console.log(JSON.stringify(result, null, 2));
+  }).catch(function (error) {
+    console.log('Error:', error.stack);
+  });
+
+};
+
+main();
```

Comment:
Maybe this runs in `npm test` by accident?

Reply:
nope, as we have ` tap test/*.test.js` in `package.json`